evm/src/cpu/kernel/asm/fields/fp6_macros.asm

```diff
@@ -0,0 +1,316 @@
+// cost: 6 loads + 6 dup/swaps + 5 adds = 6*4 + 6*1 + 5*2 = 40
+%macro load_fp6
+    // stack: offset
```

Comment:
I wouldn’t call it an `offset`. Pointer or address are better.

An offset is the difference between two addresses. So `foo + 4` is offset by `4` from `foo`, but `foo + 4` is not itself an offset.

In other words: if it makes sense to add it to a pointer, _then_ it’s an offset.